### DISCUSSION TITLE
List of supported ciphers and algorithms

### DISCUSSION THREAD

**Discussion by Nanork** [sshnet/SSH.NET#1444] [Q&A] [answered]
Is there a list of all supported ciphers and algorithms for quick reference? I can see the release notes mentioning what has been added in which version, but I would like to see a complete list of what is supported or not.
Does this exist somewhere, or could it be created?

**Comment by Rob-Hague** [comment, marked as answer]
It's on the front page if you scroll down https://github.com/sshnet/SSH.NET
